kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableConfig.java

```diff
@@ -272,6 +272,33 @@ public class TableConfig<T> {
           "needs to be a boolean.",
           true);
 
+  /**
+   * The value that enables uniform exports to Iceberg for {@linkplain
+   * #UNIVERSAL_FORMAT_ENABLED_FORMATS}.
+   *
+   * <p>{@link #ICEBERG_COMPAT_V2_ENABLED but also be set to true} to fully enable this feature.
+   */
+  public static final String UNIVERSAL_FORMAT_ICEBERG = "iceberg";
+
+  /**
+   * The value to use to enable uniform exports to Iceberg for {@linkplain
+   * #UNIVERSAL_FORMAT_ENABLED_FORMATS}.
+   */
+  public static final String UNIVERSAL_FORMAT_HUDI = "hudi";
+
+  private static final Collection<String> ALLOWED_UNIFORM_FORMATS =
+      Collections.unmodifiableList(Arrays.asList(UNIVERSAL_FORMAT_HUDI, UNIVERSAL_FORMAT_ICEBERG));
+
+  /** Table config that allows for translation of Delta metadata to other table formats metadata. */
+  public static final TableConfig<Set<String>> UNIVERSAL_FORMAT_ENABLED_FORMATS =
+      new TableConfig<>(
+          "delta.universalFormat.enabledFormats",
+          null,
+          TableConfig::parseStringSet,
+          value -> value.stream().allMatch(ALLOWED_UNIFORM_FORMATS::contains),
```

Comment:
do we want to be case sensitive?

Reply:
I haven't looked closely but I thought we were mostly case sensitive?  I'm happy to make this case insensitive if you prefer.